src/yml/normalize_yml.py

```diff
@@ -36,13 +36,20 @@
 Software = list[dict[str, Any]]
 
 
-def munge(filenames: list[str]) -> Software:
-    """Munge together the "software" nodes from YAML files into a single Python dictionary."""
+def munge(
+    filenames: list[str], owners: list[dict[str, str]] = [], canonical=False
+) -> Software:
+    """Munge together the "owners" and "software" nodes from YAML files into a single Python dictionary."""
```

Comment:
I don't think that "from YAML files" applies to the `owners` information, but the current comment makes it sound like that.  I suggest increasing clarity with something like this: 
```suggestion
    """Combine into a single Python dictionary the owner and software information, reading in the latter from YAML files."""
```

Reply:
The `owners` information is pulled in from a top-level key in the YAML file. Any `owners` keys in processed YAML files are collated and de-duplicated into one list.

Reply:
Ah, OK.